src/entry.c

```diff
@@ -1483,3 +1484,121 @@ void
 unlock_entry( const struct stumpless_entry *entry ) {
   config_unlock_mutex( entry->mutex );
 }
+
+
+
+
+static size_t append_key_value_pair(char *dest, size_t offset, const char *key, size_t key_length, const char *value, size_t value_length) {
```

Comment:
Add a documentation comment above the function definition describing what it does.